_maps/map_files/Aetherwhisp/Aetherwhisp2.dmm

```diff
@@ -10478,8 +10481,12 @@
 /turf/open/floor/plating,
 /area/maintenance/department/science/central)
 "hbD" = (
-/obj/item/wallframe/airalarm,
-/obj/item/electronics/airalarm,
+/obj/structure/disposalpipe/segment{
+	dir = 4
+	},
+/obj/machinery/airalarm/all_access{
+	pixel_y = 25
+	},
```

Comment:
Please modify this alarm (or use a preset subtype) to not send out an atmospheric alert at a 0kPa atmosphere